src/app/security-group/sg-filter/sg-filter.component.ts

```diff
@@ -44,4 +44,18 @@ export class SgFilterComponent {
   public get SecurityGroupViewMode() {
     return SecurityGroupViewMode;
   }
+
+  public ngOnChanges(changes: SimpleChanges) {
+    const accounts = changes['accounts'];
+    if (accounts) {
+      this.onAccountQueryChanged(this.accountQuery);
+    }
+  }
+
+  public onAccountQueryChanged(accountQuery: string) {
+    const queryLower = accountQuery && accountQuery.toLowerCase();
+    this.accountsFiltered = this.accounts.filter(
```

Comment:
`src/app/security-group/sg-filter/sg-filter.component.ts(57,10): error TS2339: Property 'accountsFiltered' does not exist on type 'SgFilterComponent'.`